src/config.ts

```diff
@@ -54,4 +56,37 @@ export default function config(network: Network | null) {
   };
 }
 
-export const expiryDate = process.env.REACT_APP_EXPIRY_DATE || "Jun 30 2021";
+export const expiryDate =
+  process.env.REACT_APP_EXPIRY_DATE || "Wed Jun 30 2021 22:00:00 GMT";
+export const SUPPORTED_NETWORK_IDS = [1, 42] as const;
+
+const getMerkleDistributorAddress = (chainId: 1 | 42) =>
+  chainId === 1
```

Comment:
@daywiss I actually think this is _usually_ fine since the type guarantees it's only one of those values. However, we do an unsafe case to `1 | 42` when calling this method, which means it could actually receive a value that _doesn't_ match this type. I think the runtime check that the real network ID is either 1 or 42 should happen upstream of this method (ideally where the network id is pulled in and/or defaulted) and no unsafe cast should be necessary. I think there's another thread about this, so I'll leave a quick comment there.